pkg/test/store/postgres/test_container.go

```diff
@@ -153,3 +157,46 @@ func (v *PostgresContainer) Config(dbOpts ...DbOption) (*pg_config.PostgresStore
 	GinkgoLogr.Info("Database successfully created and started", "name", cfg.DbName)
 	return cfg, nil
 }
+
+func (v *PostgresContainer) PrintDebugInfo(exportDbName string, expectedDbPort int) {
+	logEntries := make([]string, 0)
+	container := v.container
+	state, _ := container.State(context.TODO())
+	logEntries = append(logEntries, "db container details:")
+	logEntries = append(logEntries, fmt.Sprintf("trying to connect db %s on port %d, latest container: %s",
+		exportDbName, expectedDbPort, container.GetContainerID()))
```

Comment:
Kind of a nit but I find this function somewhat difficult to parse, maybe it's the style? I dunno

```suggestion
	container := v.container
	logEntries := string[]{
		"db container details:",
		fmt.Sprintf("trying to connect db %s on port %d, latest container: %s", exportDbName, expectedDbPort, container.GetContainerID()),		
	}
```

much easier to read

Reply:
Thanks for the suggestions, I'll push a new commit and apply these.